packages/shared/src/components/search/SearchBar.tsx

```diff
@@ -0,0 +1,269 @@
+            <div className="flex-1 text-theme-label-tertiary">
+              {placeholder}
+            </div>
+          )}
+
+          {sidebarRendered && (
+            <FieldInput
+              disabled={disabled}
+              placeholder={placeholder}
+              name={name}
+              id={inputId}
+              ref={inputRef}
+              onFocus={(event) => {
+                onFocus();
+                externalOnFocus?.(event);
+              }}
+              onBlur={(event) => {
+                onBlur();
+                externalOnBlur?.(event);
+              }}
+              onInput={onInput}
+              autoFocus={autoFocus}
+              type="primary"
+              aria-describedby={describedBy}
+              autoComplete="off"
+              className={classNames(
+                'flex-1 caret-theme-status-cabbage',
+                getFieldFontColor({ readOnly, disabled, hasInput, focused }),
+              )}
+              required
+            />
+          )}
+
+          <div className="flex gap-3 items-center">
+            {hasInput && (
+              <Button
+                {...rightButtonProps}
+                className="btn-tertiary"
+                buttonSize={ButtonSize.Small}
+                title="Clear query"
+                onClick={onClearClick}
+                icon={<CloseIcon />}
+                disabled={!hasInput}
+              />
+            )}
+
+            {sidebarRendered && (
+              <div className="h-8 border border-theme-divider-quaternary" />
+            )}
+
+            <SimpleTooltip
+              content={
+                searchHistory.length === 0
+                  ? 'Your search history is empty'
+                  : 'See search history'
+              }
+            >
+              <div>
+                <Button
+                  {...rightButtonProps}
+                  className="btn-tertiary"
+                  buttonSize={ButtonSize.Small}
+                  title="Search history"
+                  onClick={seeSearchHistory}
+                  icon={<TimerIcon />}
+                  disabled={searchHistory.length === 0}
+                />
+              </div>
+            </SimpleTooltip>
+            {sidebarRendered && (
+              <SimpleTooltip
+                content={!hasInput && 'Enter text to start searching'}
+              >
+                <div>
+                  <Button
+                    {...rightButtonProps}
+                    className="btn-primary"
+                    title="Submit"
+                    onClick={onSubmit}
+                    icon={<SendAirplaneIcon size={IconSize.Medium} />}
+                    disabled={!hasInput}
+                  />
+                </div>
+              </SimpleTooltip>
+            )}
+          </div>
+        </BaseField>
+
+        <RaisedLabel type={RaisedLabelType.Beta} />
+      </RaisedLabelContainer>
+
+      {sidebarRendered && showProgress && (
```

Comment:
Not sure all these sidebarRendered evaluations are correct actually
Don't we also show this on mobile? (Same question for multiple items here)